### PR TITLE
Create a function that verifies the correctness of the answer

### LINKED ISSUE #7: Logika | Weryfikowanie poprawności odpowiedzi #23
### Kryteria akceptacji

Należy zweryfikować i skonsultować, a także ewentualnie uaktualnić przed rozpoczęciem prac nad zgłoszeniem.

- [ ]  Funkcja przyjmuje odpowiedź prawidłową i odpowiedź udzieloną przez gracza (człowieka lub komputer).

- [ ]  Funkcja zwraca informację czy odpowiedź jest prawidłowa, czy błędna.

- [ ]  Wewnątrz funkcji należy zaproponować i zaimplementować algorytm decydujący o poprawności odpowiedzi.

- [ ] Działanie komponentu pokryte testami jednostkowymi.

### PR DESCRIPTION
**Pull request by KlaudiaDabrowska**
closes #7

### CODE REVIEW

**Review by Smyku6 — COMMENTED** [review]
If function return only Boolean values we can shorten that function:
`const isAnswerCorrect = (correctAnswer, chosenAnswer) => correctAnswer === chosenAnswer;`

File: src/app/logic/correctAnswer.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+const isAnswerCorrect = (correctAnswer, chosenAnswer) => {
```
Comment (vieraboschkova) [inline src/app/logic/correctAnswer.js:1]: Perhaps you could name the file exactly the same as the name of the function?

File: src/app/logic/correctAnswer.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+  return correctAnswer === chosenAnswer;
+};
+
```
Comment (vieraboschkova) [inline src/app/logic/correctAnswer.js:4]: If you want to use this function outside of this file, you will need to add export to it.

File: src/app/logic/correctAnswer.js (hunk 1 above)
Comment (KlaudiaDabrowska) [reply src/app/logic/correctAnswer.js:1]: Yeah, you are right.

File: src/app/logic/correctAnswer.js (hunk 2 above)
Comment (KlaudiaDabrowska) [reply src/app/logic/correctAnswer.js:4]: Oh god, I forgot about it...

**Review by Smyku6 — APPROVED** [review]

**Pull request merged**